src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -475,6 +481,38 @@ export function fetchRequestDetails(id) {
   };
 }
 
+export function fetchClaim(appointmentId, appointment) {
+  return async dispatch => {
+    try {
+      if (!appointment.claimId) {
+        dispatch({
+          type: FETCH_CLAIM,
+        });
+        const { claimId } = await getClaim(appointment.start);
+        dispatch({
+          type: FETCH_CLAIM_SUCCEEDED,
+          id: appointmentId,
+          claimId,
+        });
+      }
+    } catch (e) {
+      if (e.issue.code === '404') {
```

Comment:
are you thinking that vets-api is going to send a 404 when there is no claim found? I feel like they would sanitize the expected not found response